crates/uv/src/commands/venv.rs

```diff
@@ -142,7 +141,7 @@ async fn venv_impl(
     exclude_newer: Option<ExcludeNewer>,
     concurrency: Concurrency,
     native_tls: bool,
-    no_config: bool,
+    _no_config: bool,
```

Comment:
Remove?

Reply:
Good point flagging this. We need to decide if `--no-config` should continue to omit `.python-version` file discovery. I'm not sure it does consistently today, but it did here. I think I dropped because it's confusing — but we should discuss that choice.

Reply:
There are a bunch of places today where we don't handle this. For example, `uv export` doesn't even have a `--no-config` flag.